docs/get-started/dapps/reach.md

```diff
@@ -0,0 +1,446 @@
+title: Build with Reach
+
+Alice and Bob are now ready to kick-off development of their Algorand-powered auction dApp! This guide will take you through the steps to build the dApp, which includes setting up a development environment, writing the dApp code which can contain multiple smart contracts, deploying it, and writing the functions to interact with it.
+
+A few tips before getting started. The goal of this guide is to get you up and running with a working prototype that represents a real use case, as quickly as possible. We use a hands-on example to teach you basic design principles and best practices for building dApps with Reach on Algorand. This guide does _not_ cover all the details of the dApp backend and frontend code. This is intentional so that you can focus on solidifying higher-level concepts that will be the foundation for building _any_ dApp on Algorand using Reach. So don’t worry if you don’t understand what everything does in the solution. This is expected! After you feel comfortable with the basics, you can head over to the detailed Reach documentation and work on becoming an expert in the Reach dApp language.
+
+Now let’s get started. 
+
+# Organization
+
+This guide is organized into two sections. The first section helps you launch the dApp and run an auction simulation. The second section provides a deeper dive into the different components of the auction application.
+
+All of the code for this guide is located here for [Auction using Reach](https://github.com/algorand/reach-auction). The  `index.mjs` and `index.rsh` files are included, so follow along!
+
+## Install Reach
+
+Reach is designed to work on POSIX systems with [make](https://en.wikipedia.org/wiki/Make_(software)), [Docker](https://www.docker.com/get-started), and [Docker Compose](https://docs.docker.com/compose/install/) installed. The best way to install Docker on Mac and Windows is with [Docker Desktop](https://www.docker.com/products/docker-desktop). 
+
+
+To confirm everything is installed try to run the following three commands and see no errors.
+
+
+```
+  $ make --version
+  $ docker --version
+  $ docker-compose --version
+```
+
+
+If you’re using Windows, consult [the guide to using Reach on Windows](https://docs.reach.sh/guide-windows.html).
+
+Once confirmed that reach prerequisite are installed, choose a directory for this project such as:
+
+```
+  $ mkdir -p ~/reach && cd ~/reach
+```
+
+Navigate to the project folder
+
```

Comment:
Eliminate this section? Above you created the ~/reach directory and changed directory to it. Should be all set to install from here.

Reply:
good catch.. moved the clone up ... all directories line up now.